src/common/CommonUtils.ts

```diff
@@ -74,29 +71,6 @@ export class CommonUtils {
         });
     }
 
-    public static async createTempDirectory(
```

Comment:
As I stated offline, I think we should keep this. It's the right pattern for guaranteeing a unique working directory.

I think the only thing we really need to do here is kill the input parameter, and just use a literal for the folder prefix. It's our code, after all, and we can create the folder we want.  If we instead do:
```typescript
const mkdtemp = util.promisify(fs.mkdtemp);
const folderPrefix = 'lwc-mobile-';
const tempFolderPath = path.join(os.tmpdir(), folderPrefix);
return mkdtemp(tempFolderPath) // ...
```
We're going to get temp directories created like `/var/folders/ph/x5q98_c53kv31x739xcx2z5c0000gp/T/lwc-mobile-9jjMpW` on macOS, `/tmp/lwc-mobile-9jjMpW` on Linux, and `C:\Users\...\AppData\Local\Temp\lwc-mobile-9jjMpW` on Windows.

This is what we want. It's the right pattern for creating a disposable working directory. We should just fix what's broken in our code.